packages/dcos-integration-test/extra/test_cockroach_config.py

```diff
@@ -0,0 +1,39 @@
+"""
+Testing the CockroachDB setup in DC/OS.
+"""
+import logging
+import subprocess
+
+from retrying import retry
+
+
+__maintainer__ = 'tweidner'
+__contact__ = 'security-team@mesosphere.io'
+
+
+LOG = logging.getLogger(__name__)
+
+
+def test_config_change_applied():
+    config_change_unit = 'dcos-cockroachdb-config-change'
+    subprocess.check_call(
+        ['sudo', 'systemctl', 'restart', config_change_unit],
```

Comment:
sudo?

Reply:
Strange - but I guess we assume that the tests run as a user with sudo privileges.

Reply:
Pre-cautious to run on AWS, not sure whether it is required.